app/controllers/system_status_controller.rb

```diff
@@ -99,20 +88,20 @@ def details
 
     payload = {
       db: db_message,
-      jobs_stats: jobs_stats,
-      jobs_message: jobs_message,
-      revision: revision.chomp,
-      branch: branch,
+      jobs_stats:,
+      jobs_message:,
+      revision: Git.revision,
+      branch: Git.branch,
       hostname: `hostname`.chomp,
       cache: cache_message,
       user_count_positive: User.all.any?,
       registered_deployment_id: Rails.cache.read('registered-deployment-id'),
       environment_deployment_id: ENV['DEPLOYMENT_ID'],
       last_migration: {
-        cas: cas,
+        cas:,
```

Comment:
Something's missing.

Reply:
You'd think.  That's ruby's new hotness in 3.1.  I'm not 100% sold on it, but rubocop is. https://blog.saeloun.com/2021/09/28/ruby-allow-value-omission-in-hash-literals.html

Reply:
My professional opinion on that syntax is that it stinks. It's like `<blink>` or `goto`. I'm 0% sold on it :-)

Reply:
Agreed.  Updating .rubocop.yml now